### ISSUE TITLE
Properties with new lines can create invalid SDFiles

### ISSUE THREAD

**Issue by bp-kelley** [rdkit/rdkit#5827]
**Describe the bug**
Adding a string property with multiple new lines generates invalid SD blocks.

**To Reproduce**
```
>>> m.SetProp("foo", "docked\n\ndocked")
>>> out = Chem.SDWriter(sys.stdout)
>>> out.write(m)
>>> out.close()

     RDKit          2D

  1  0  0  0  0  0  0  0  0  0999 V2000
    0.0000    0.0000    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
M  END
>  <foo>  (1) 
docked

docked

$$$$
```

**Expected behavior**
RDKit should either reject writing these properties or strip the new lines.  It shouldn't write an invalid file.

**Comment by greglandrum** [comment]
It's easy enough to fix this. The question is what the right behavior is:
1. Report error and fail to write the molecule
2. Report warning and skip the property for that molecule
3. Report warning and just write the first line
4. Report warning and somehow modify the property so that it doesn't contain the `\n`

I would not do either 1 or 4, but I think either 2 or 3 would be ok. I guess I have a slight preference for 2.

**Comment by greglandrum** [comment]
Ok, I implemented solution 2 in the fix in #5873
